contracts/AloeBlend.sol

```diff
@@ -594,8 +594,13 @@ contract AloeBlend is AloeBlendERC20, UniswapHelper, IAloeBlend {
         uint256[14] storage array = gasPriceArrays[_token];
         uint8 idx = gasPriceIdxs[_token];
         unchecked {
+            uint256 average = gasPrices[_token];
+            uint256 maxDelta = average / 10;
```

Comment:
Add comment: constrain movement of new entry to +- 10% of previous average value